### DISCUSSION TITLE
Training data

### DISCUSSION THREAD

**Discussion by erees1** [snakers4/silero-vad#78] [Q&A] [answered]
Thanks for the interesting repo and models! I wondered if you are able to give some more information on what data was used to train the system? Is it publicly available data? Are you able to share the data?

Thanks!

**Comment by snakers4** [comment, marked as answer]
Hi,

- The VAD is trained on the downstream data of our STT systems, for other models it is a bit more complicated
- No
- Partially yes, but we have no intentions to do so
